lib/sonos.js

```diff
@@ -719,7 +719,7 @@ Sonos.prototype.getCurrentState = function(callback) {
  * Search "Class"
  * Emits 'DeviceAvailable' on a Sonos Component Discovery
  */
-var Search = function Search() {
+var Search = function Search(timeout) {
```

Comment:
I'd rather this be an options object e.g `{ timeout: 10 * 1000 }` so we can add other arguments to the search functionality later without a crazy amount of arguments to the function.